webapp/channels/src/i18n/en.json

```diff
@@ -5457,15 +5457,18 @@
   "user.settings.notifications.header": "Notifications",
   "user.settings.notifications.icon": "Notification Settings Icon",
   "user.settings.notifications.info": "Desktop notifications are available on Edge, Firefox, Safari, Chrome and Mattermost Desktop Apps.",
-  "user.settings.notifications.keywordsWithHighlight.disabledTooltipTitle": "This feature is available on the Professional plan",
+  "user.settings.notifications.keywordsWithHighlight.disabledTooltipMessage": "This feature is available on the Professional plan",
+  "user.settings.notifications.keywordsWithHighlight.disabledTooltipTitle": "Professional feature",
   "user.settings.notifications.keywordsWithHighlight.extraInfo": "These keywords will be shown to you with a highlight when anyone sends a message that includes them.",
   "user.settings.notifications.keywordsWithHighlight.inputTitle": "Enter non case-sensitive keywords, press Tab or use commas to separate them:",
   "user.settings.notifications.keywordsWithHighlight.none": "None",
   "user.settings.notifications.keywordsWithHighlight.professional": "Professional",
   "user.settings.notifications.keywordsWithHighlight.title": "Keywords That Get Highlighted (without notifications)",
-  "user.settings.notifications.keywordsWithHighlight.userModal.messageAdminPreTrial": "Create unlimited keywords with highlight with one of our paid plans. Get the full experience of Enterprise when you start a free, {trialLength} day trial.",
+  "user.settings.notifications.keywordsWithHighlight.userModal.messageAdminPostTrial": "Get the ability to passively highlight keywords that you care about. Upgrade to the Professional plan to create unlimited highlight keywords.",
+  "user.settings.notifications.keywordsWithHighlight.userModal.messageAdminPreTrial": "Get the ability to passively highlight keywords that you care about. Upgrade to the Professional plan to create unlimited highlight keywords.",
```

Comment:
nit: Do we need to translations strings if they are exactly the same?

Reply:
yeah for now i have kept these different if @abhijit-singh have any variation to be done there